CHANGES.md

```diff
@@ -39,6 +39,9 @@ unreleased
 - Fix in `Location`: make `raise_errorf` exception equivalent to exception
   `Error` (#242, @pitag-ha)
 
+- Fix in `Pprintast`: correctly pretry print local type substitutions, e.g. type
+  t := ... (#261, @matthewelse)
```

Comment:
> hopefully the change log entry is sufficient

Yes, it is, thanks! There's just a typo I'm suggesting to remove. And the line break is to have the code example all in one line, but we can also leave it as is

```suggestion
- Fix in `Pprintast`: correctly pretty print local type substitutions, e.g.
  type t := ... (#261, @matthewelse)
```